### PR TITLE
Start / Stop all 

### PR DESCRIPTION
**Pull request by RomiRand**
Added buttons for starting/stopping the entire stack.

### CODE REVIEW

File: src/react/commands/RocketPool.ts [hunk 1]
```diff
@@ -40,6 +40,32 @@ const getEth2ClientName = (): string => {
   }
 }
 
+const stopRocketPool = async() => {
```
Comment (colfax23) [inline src/react/commands/RocketPool.ts:43]: Nice!  Reading this I was wondering why I hadn't already written these, and it looks like I did below (however I dont call them during the install).  If you have a chance you can use the below implementation of startNodes and stopNodes instead.

File: src/react/commands/RocketPool.ts [hunk 2]
```diff
@@ -97,17 +123,11 @@ const installAndStartRocketPool = async (callback: Callback, stdoutCallback: Std
   }
 
   // Just in case nodes were running - pick up new config (might happen anyway on start, not sure)
-  const stopNodesRc = await executeCommandStream(wrapCommandInDockerGroup(ROCKET_POOL_EXECUTABLE + " service stop -y"), internalStdoutCallback);
-  if (stopNodesRc != 0) {
-    console.log("stop nodes failed");
-    callback(false);
-  }
+  // await stopRocketPool(callback, stdoutCallback);
+  await stopRocketPool();
```
Comment (colfax23) [inline src/react/commands/RocketPool.ts:127]: Per the above comment, lets call the below implementation here :)

File: src/react/pages/Status.tsx [hunk 3]
```diff
@@ -8,11 +8,24 @@ import {
...
+  queryEth2BeaconStatus,
+  queryEth2ValidatorStatus,
+  startRocketPool,
+  stopRocketPool
+} from '../commands/RocketPool'
 
 import Footer from "../components/Footer";
 import { shell } from "electron";
 import styled from "styled-components";
+import { Link } from 'react-router-dom'
```
Comment (colfax23) [inline src/react/pages/Status.tsx:28]: Mistaken import?

File: src/react/pages/Status.tsx [hunk 4]
```diff
@@ -145,6 +181,11 @@ const Status = () => {
     return (
       <ResultsTable>
         <thead>
+          <tr>
+            <th colSpan='2'/>
+            <th><StartButton onClick={startRocketPool}>Start All</StartButton></th>
```
Comment (colfax23) [inline src/react/pages/Status.tsx:186]: Awesome - so excited to see this :)

File: webpack.react.config.js [hunk 5]
```diff
@@ -9,6 +9,7 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 module.exports = {
   // development mode will set some useful defaults in webpack
   mode: 'development',
+  devtool: 'source-map',
```
Comment (colfax23) [inline webpack.react.config.js:12]: Do you know what this is for?

**Review by colfax23 — CHANGES_REQUESTED** [review]
Hey - thanks for submitting!  This will definitely be helpful.  Added some comments in case you have some time to revisit.  Let me know if you have any questions.

File: src/react/commands/RocketPool.ts (hunk 1 above)
Comment (RomiRand) [reply src/react/commands/RocketPool.ts:50]: Whoops, sure. Fixed.

File: src/react/pages/Status.tsx (hunk 3 above)
Comment (RomiRand) [reply src/react/pages/Status.tsx]: Probably

File: webpack.react.config.js (hunk 5 above)
Comment (RomiRand) [reply webpack.react.config.js]: I played around with debugging and this was needed for some reason. Shouldn't be in release code though

File: src/react/commands/RocketPool.ts (hunk 1 above)
Comment (colfax23) [reply src/react/commands/RocketPool.ts:50]: Awesome!  Can you remove these duplicate implementations?

File: src/react/pages/Status.tsx [hunk 6]
```diff
@@ -76,6 +88,33 @@ const LogsButton = styled.button`
   }
 `;
 
+
+const StartButton = styled.button`
```
Comment (colfax23) [inline src/react/pages/Status.tsx:92]: Can we reuse the style of the LogsButton so they look similar?  Also, I noticed that when the text changes the button size changes, which alters the spacing of the whole table.  Can you make it fixed so it doesnt change size?

File: src/react/pages/Status.tsx [hunk 7]
```diff
@@ -141,10 +210,25 @@ const Status = () => {
     return eth2ValidatorContainerStatus;
   }
 
+  const computeTotalStatus = () => {
+    if (computeEth1Status() != 2 && eth2BeaconContainerStatus != 2 && eth2ValidatorContainerStatus != 2)
```
Comment (colfax23) [inline src/react/pages/Status.tsx:214]: nit: { }

File: src/react/pages/Status.tsx [hunk 8]
```diff
@@ -141,10 +210,25 @@ const Status = () => {
...
+    // some running, some stopped
+    return 3
+  }
+
   const renderNodeStatusTable = () => {
     return (
       <ResultsTable>
         <thead>
+          <tr>
+            <th colSpan='3'/>
```
Comment (colfax23) [inline src/react/pages/Status.tsx:229]: I ran into a build error here, I think it should be {3} instead of '3'.  Does this work for you?

**Review by colfax23 — CHANGES_REQUESTED** [review]
Looks great!  So close, just a couple small things.

**Review by colfax23 — APPROVED** [review]
Looks great - thanks!

**Pull request merged**